BC-Gov-Org-HowTo/Cheatsheet.md

```diff
@@ -0,0 +1,76 @@
+## "Working in github.com/bcgov" Cheatsheet
+
+This github.com/bcgov Cheatsheet covers:
+
+- a few things to help when you are just starting out
+- things to touch base with *every time* you open or contribute to repository (as a BC Government employee)
+
+ All BC Government employees working in github.com/bcgov should be familiar with the full [BC-Policy-Framework-For-GitHub](https://github.com/bcgov/BC-Policy-Framework-For-GitHub)
+
+
+### Joining github.com/bcgov
+- Create a GitHub account with your work email and "Province of British Columbia" as the company (optional) _and_ join the [BCGov GitHub Organization](https://github.com/bcgov)
```

Comment:
The instructions are to e-mail using the address on the main bcgov org page, which is @kelpisland